web/src/routes/(user)/tags/[[photos=photos]]/[[assetId=id]]/+page.ts

```diff
@@ -0,0 +1,38 @@
+import { QueryParameter } from '$lib/constants';
+import { authenticate } from '$lib/utils/auth';
+import { getFormatter } from '$lib/utils/i18n';
+import { getAssetInfoFromParam } from '$lib/utils/navigation';
+import { buildTree, normalizeTreePath } from '$lib/utils/tree-utils';
+import { getAllTags, type AssetResponseDto } from '@immich/sdk';
+import type { PageLoad } from './$types';
+
+export const load = (async ({ params, url }) => {
+  await authenticate();
+  const asset = await getAssetInfoFromParam(params);
+  const $t = await getFormatter();
+
+  let pathAssets: AssetResponseDto[] = [];
+  const path = url.searchParams.get(QueryParameter.PATH);
+  if (path) {
+    pathAssets = [];
+  }
```

Comment:
What is happening here? `pathAssets` is already an empty array at this point

Reply:
Probably not needed anymore.